src/System.Net.Http/tests/FunctionalTests/HttpClientHandlerTest.cs

```diff
@@ -385,11 +385,31 @@ public void MaxRequestContentBufferSize_Set_ThrowsPlatformNotSupportedException(
                     Assert.Equal(HttpStatusCode.OK, httpResponse.StatusCode);
                     string responseText = await httpResponse.Content.ReadAsStringAsync();
                     _output.WriteLine(responseText);
-                    Assert.True(TestHelper.JsonMessageContainsKeyValue(responseText, "Cookie", name + "=" + value));
+                    Assert.True(TestHelper.JsonMessageContainsKeyValue(responseText, cookieName, cookieValue));
                 }
             }
         }
 
+        [ActiveIssue(4960, PlatformID.AnyUnix)]
+        [Theory]
+        [InlineData("cookieName1", "cookieValue1")]
+        public async Task GetAsync_RedirectResponseHasCookie_CookieSentToFinalUri(string cookieName, string cookieValue)
+        {
+            Uri uri = HttpTestServers.RedirectUriForDestinationUri(false, HttpTestServers.RemoteEchoServer, 1);
```

Comment:
Please add more details to what this does. What's "false" and 1 in this context? (Maybe you could use the parameter names within the call to make things a bit clearer.)